nilearn/plotting/displays.py

```diff
@@ -873,8 +923,7 @@ def annotate(self, left_right=True, positions=True, size=12, **kwargs):
         size: integer, optional
             The size of the text used.
         kwargs:
-            Extra keyword arguments are passed to matplotlib's text
-            function.
+            Extra keyword arguments are passed to matplotlib.
```

Comment:
I really don't like it that it is now unclear where the kwargs are sent. It makes them very hard to use.